DependencyInjection/Configuration.php

```diff
@@ -65,6 +65,12 @@ public function getConfigTreeBuilder(): TreeBuilder
         $this->addDbalSection($rootNode);
         $this->addOrmSection($rootNode);
 
+        $rootNode
+            ->children()
+            ->booleanNode('use_trigger_error_for_deprecations')
+            ->defaultFalse()
```

Comment:
Will this default change in 3.0.0? If yes, should we deprecate not configuring this node explicitly?
As a follow-up, I think a contribution should be made to the Symfony recipe, setting it to `true`.

Reply:
I don’t see why it should change?